Emscripten-build-instructions.md

```diff
@@ -0,0 +1,167 @@
+# Wasm Build Instructions
+
+## CppInterOp Wasm Build Instructions
+
+This document first starts with the instructions on how to build a wasm  
+build of CppInterOp. Before we start it should be noted that  
+unlike the non wasm version of CppInterOp we currently only  
+support the Clang-REPL backend using llvm>19 for osx and Linux.  
+We will first make folder to build our wasm build of CppInterOp.  
+This can be done by executing the following command
+
+```bash
+mkdir CppInterOp-wasm
+```
+
+Now move into this directory using the following command
+
+```bash
+cd ./CppInterOp-wasm
+```
+
+To create a wasm build of CppInterOp we make use of the emsdk  
+toolchain. This can be installed by executing (we only currently  
+support version 3.1.45)
+```bash
+git clone https://github.com/emscripten-core/emsdk.git
+cd emsdk
+./emsdk install  3.1.45
+```
+
+Go back upto the top level build directory using
+
+```bash
+cd ../..
+```
+
+and activate the emsdk environment
+```bash
+./emsdk/emsdk activate 3.1.45
+source ./emsdk/emsdk_env.sh
+```bash
+
+Now clone the 19.x release of the LLVM project repository 
+and CppInterOp
+
+```bash
+git clone --depth=1 --branch release/19.x https://github.com/llvm/llvm-project.git
```

Comment:
![image](https://github.com/user-attachments/assets/577d6e57-19c4-440d-aafc-dc0b3a43a1f6)

This looks weird in preview.